### DISCUSSION TITLE
Remote stop and restart during an outage.

### DISCUSSION THREAD

**Discussion by simont65** [jgyates/genmon#1101] [Questions] [answered]
Hello, I just had my first power outage last week. The power was out for about 40 hours. I don’t necessarily need the generator after going to bed. To save on propane I would like to be able to shutdown the generator at night and power it back on remotely during an outage. Is it possible/feasible?

I do have a generac 22kw air cooled with an evolution 2 contrôler and a smart transfer switch.

**Comment by jgyates** [comment, marked as answer]
If you have an Generac Air cooled controller, then no. The remote stop command will only stop a manual or remote start. An automatic start can not be stopped by a remote command. This is a function of the controller, not the genmon software. Note that if you generator is running in automatic mode under heavy load you should try to remove the load first before stopping with the stop Off button as it is not good for your generator.

If you have an Evolution Liquid Cooled controller you have access to remotely setting the Auto, Manual and Off buttons via remote commands.
